browser/src/control/Control.NotebookbarBuilder.js

```diff
@@ -1065,11 +1065,12 @@ L.Control.NotebookbarBuilder = L.Control.JSDialogBuilder.extend({
 
 			var hasManyChildren = childData.children && childData.children.length > 1;
 			if (hasManyChildren) {
-				var tableId = childData.id ? childData.id.replace(' ', '') : '';
+				var tableId = childData.id ? childData.id.replace(' ', '') + ' container': String(parseInt(Math.random() * 1000));
```

Comment:
this can be problematic... we have to use exactly the same id as we receive with JSON so later when we receive actions like 'hide' we will find that element